example.js

```diff
@@ -5,7 +5,7 @@ state.set({
 })
 
 state.subscribe({
-  counter: [document.querySelector('span'), 'innerText']
+  counter: [document.querySelector('span'), 'innerHTML']
```

Comment:
innerHTML would change all node inside `span`, not that there is any for now, but still innerText is better.

Reply:
and also just live it as `'span'`. We don't need `document.querySelector('')` around it, we are selecting node inside the library. I made these changes earlier in the morning.

Reply:
Ohhh! they were the merging issues! 😲 😞